template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Views/Account/Login.cshtml

```diff
@@ -59,7 +60,8 @@
             <h4>Use another service to log in.</h4>
             <hr />
             @{
-                var loginProviders = SignInManager.GetExternalAuthenticationSchemes().ToList();
+                var schemes = await SchemeProvider.GetAllSchemesAsync();
```

Comment:
Maybe we can start by just adding a property to the AuthenticationScheme called Remote, and only set that for our Remote handlers (OAuth based + Twitter), then we can filter for Remote schemes?